src/main/java/uk/ac/cam/cl/dtg/isaac/api/PagesFacade.java

```diff
@@ -388,28 +388,29 @@ public final Response getQuestionList(@Context final Request request,
         }
 
         try {
-            if (null == statuses || Objects.equals(statuses, "")) {
-                // If no statuses apply assume all statuses
-                filterByStatuses = Set.of(CompletionState.values());
+            if (null == statuses) {
+                filterByStatuses = null;
+            } else if (statuses.isEmpty()) {
+                filterByStatuses = CompletionState.ALL_STATES;
```

Comment:
Does this need explaining, or is it obvious?

Reply:
I think a comment to explain that this is used to determine whether or not to augment the results with question attempt data would be worth it. I wouldn't say that would be expected/obvious!